src/integration/wirespec/src/jvmMain/kotlin/flock/community/wirespec/Wirespec.kt

```diff
@@ -0,0 +1,26 @@
+package community.flock.wirespec
```

Comment:
Can you explain why this file is placed here? Shouldn't this be part of the core instead? Especially since you're referring to this file in generated src code?

Reply:
This module with the Wirespec file is created to depend on it in the jackson module with compileOnly. The Wirespec file is needed to compile the jackson module but the Wirespec file is not added as a dependency because this file is generated by the plugin. This would conflict when to files are on the classpath

https://github.com/flock-community/wirespec/pull/163/files/d4c91fca9d50b36e2ebebd4c20acdc094668b89a#diff-8c443eb9e84ecab423fc2d2f972eea59206c574d5e21fa18e0f7aa1c105ccad7R27

Reply:
How can we make sure we keep this maintainable, i.e. how do we do knowledge sharing? Does each module deserve its own readme? Should we add comments to the source code. Do something FAQ-y? 

I'm afraid that we have to do something, otherwise the existence of this module would be questioned time and again.

Additional food for thought: can we have a script to update all generated content in the repository

Reply:
- Lets create a readme for this integration module
- We can write a script that generates the sources (will be documented in the readme)